pkg/reconciler/v1alpha1/revision/resources/deploy.go

```diff
@@ -111,6 +119,13 @@ func makePodSpec(rev *v1alpha1.Revision, loggingConfig *logging.Config, observab
 	userContainer.VolumeMounts = append(userContainer.VolumeMounts, varLogVolumeMount)
 	userContainer.Lifecycle = userLifecycle
 
+	// Merge user container env vars.
```

Comment:
Does the runtime contract indicate what happens here when the user has specified `PORT`?

Reply:
If it is suitably substantial, can you file an issue and `TODO(#1234)` for follow up?

Reply:
It doesn't say. But it says "The following environment variables MUST be set". Should I file an issue for the runtime or what?